int8 CAFFE_CONTROLLER = 1
int8 LIN_GAUSS_CONTROLLER = 2
int8 controller_to_execute

CaffeParams caffe
LinGaussParams lingauss